biome.json

```diff
@@ -0,0 +1,113 @@
+{
+	"$schema": "https://biomejs.dev/schemas/1.5.3/schema.json",
+	"organizeImports": {
+		"enabled": true
+	},
+	"linter": {
+		"enabled": true,
+		"rules": {
+			"recommended": true,
+			"a11y": {
+				"useKeyWithClickEvents": "off",
```

Comment:
These are all the lint rules I had to disable so that it wouldn't complain about any of our current code.

Many of them are very sensible and I think we'd do well to fix most of the things it complains about (especially in the non-style related rule categories)